pegasus/sites.v3/code.org/views/featured_project.html

```diff
@@ -2,7 +2,7 @@
   <a href="%url%" style="font-family: 'Gotham 4r', sans-serif; color: black; text-decoration: none">
     <div style="border: 2px solid rgb(187, 187, 187); border-radius: 10px; height: 270px; overflow: hidden; text-align: center">
       <div style="width: 100%; height: 85%; overflow: hidden">
-        <img src="%image%" style="width: 100%"/>
+        <img src="%image%" style="width: 100%" alt="%alt_text%"/>
```

Comment:
This is used a few other pages too so I made tickets to add alt text there as well: [A11Y-209](https://codedotorg.atlassian.net/browse/A11Y-209) and [A11Y-210](https://codedotorg.atlassian.net/browse/A11Y-210)

Reply:
Thanks for tracking this!